x-pack/plugins/security_solution/public/common/lib/cell_actions/expanded_cell_value_actions.tsx

```diff
@@ -76,6 +88,34 @@ const ExpandedCellValueActionsComponent: React.FC<Props> = ({
           />
         ) : null}
       </StyledContent>
+      {scopeId !== TimelineId.casePage && (
+        <StyledFlexGroup gutterSize="s">
+          <EuiFlexItem>
+            {timelines.getHoverActions().getFilterForValueButton({
+              Component: EuiButtonEmpty,
+              field: field.id,
+              filterManager,
+              onFilterAdded,
+              ownFocus: false,
+              size: 's',
+              showTooltip: false,
+              value,
+            })}
+          </EuiFlexItem>
+          <EuiFlexItem>
+            {timelines.getHoverActions().getFilterOutValueButton({
+              Component: EuiButtonEmpty,
+              field: field.id,
+              filterManager,
+              onFilterAdded,
+              ownFocus: false,
+              size: 's',
+              showTooltip: false,
+              value,
+            })}
+          </EuiFlexItem>
+        </StyledFlexGroup>
+      )}
```

Comment:
Hey, this part was removed on purpose, FilterIn/Out actions are already in the default column cellActions. This change makes them appear duplicated.

![duplicated_filter_actions](https://user-images.githubusercontent.com/17747913/216959649-ce0d52f6-1948-4ee2-9493-2c51abad44ab.png)

Reply:
@semd, Thank you. I will remove this code.

Reply:
Corrected.